src/features/view-subdomains/SubdomainMetrics/SubdomainMetrics.tsx

```diff
@@ -42,26 +41,23 @@ export const SubdomainMetrics: FC<SubdomainMetricsProps> = ({
 		{
 			title: 'Items In Domain',
 			value: {
-				isLoading,
+				...DEFAULT,
```

Comment:
is there a difference for using `...DEFAULT` over `isLoadingMetrics`? If not I'd probs go with using `isLoadingMetrics`

Reply:
Looks like no difference to me, I'll change to `isLoadingMetrics` for clarity.